spec/views/public_body/list.html.erb_spec.rb

```diff
@@ -0,0 +1,46 @@
+# -*- encoding : utf-8 -*-
+require File.expand_path(File.join('..', '..', '..', 'spec_helper'), __FILE__)
+
+describe "public_body/list" do
+  before do
+    @pb = mock_model(PublicBody,
+      :name => 'Test Quango',
+      :short_name => 'tq',
+      :url_name => 'testquango',
+      :notes_without_html => '',
+      :created_at => Time.now,
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.